bakerydemo/settings/base.py

```diff
@@ -13,6 +13,7 @@
 import os
 
 import dj_database_url
+from django.utils.csp import CSP
```

Comment:
Is it possible for us to move this and the middleware addition within a Django version check, so we can merge this while still supporting Django 5.2?